javascript/src/utils/stream.utils.ts

```diff
@@ -164,10 +164,7 @@ export function guessDeltaIndex(
       } else {
```

Comment:
This else technically becomes unnecessary in this case

Reply:
i will make that change on the hoangvvo/llm-sdk and include on next release